static/js/tools/statsvar_menu.tsx

```diff
@@ -182,6 +191,13 @@ class Menu extends Component<MenuPropType, {}> {
       </div>
     );
   }
+  componentDidMount(){
+    $.getJSON(jsonPath, function(data){
```

Comment:
Can also use axios for consistency. the can remove jquery from package. json

Reply:
done.